rosetta-runtime/src/main/java/com/rosetta/model/lib/meta/RosettaMetaData.java

```diff
@@ -33,9 +33,20 @@ public interface RosettaMetaData<T extends RosettaModelObject> {
 	
 	List<Function<? super T, QualifyResult>> getQualifyFunctions(QualifyFunctionFactory factory);
 	
+	default Validator<? super T> validator(ValidatorFactory factory) {
+		return null;
```

Comment:
Instead of returning null here, could you please return the deprecated `validator()`? That'll make integration a bit easier - as well as the eventual removal of the deprecated methods.